regression/cbmc-java/dynamic-multi-dimensional-array/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+TestClass.class
+--function TestClass.f --cover location --unwind 2
+Source GOTO statement: .*
+(^ exception: Can't convert byte_extraction|warning: Nested exception printing not supported on Windows)
```

Comment:
The output doesn't actually include "warning: ", hence this currently fails on Windows/AppVeyor.